docs/sdks/client-sdks/flutter/quickstart.mdx

```diff
@@ -0,0 +1,290 @@
+
+```shell
+flutter pub get
+```
+
+## Feature Flags
+
+Feature flags are a way to toggle features on and off without needing to deploy code.
+
+### Initialize the SDK
+
+[Create an SDK key](/sdks/sdk-keys) if you don't already have one.
+
+```dart
+import 'package:eppo/eppo.dart';
+
+// Create subject
+final subject = Subject(
+  subjectKey: subjectKey,
+  subjectAttributes: ContextAttributes(),
+);
+
+// Create precompute arguments
+final precomputeArgs = PrecomputeArguments(subject: subject);
+
+// Create SDK options
+final sdkOptions = SdkOptions(
+  sdkKey: sdkKey
+);
+
+// Create client
+Eppo.initialize(precomputeArgs, sdkOptions);
+```
+
+:::note
+The SDK key is different from the project API key. You can find your SDK key in the [SDK Keys section of the Eppo interface](https://eppo.cloud/configuration/environments/keys).
+:::
+
+### How assignments work
+
+On session start, the SDK retrieves precomputed evaluation of configuration rules from the Eppo Edge server that define how your subject should be allocated to variants.
+The configuration is cached locally; when you call an assignment function, the SDK evaluates these rules locally without making additional network requests.
+
+Initializing the SDK requires:
+
+- **SDK Key**: Authorizes the SDK to connect to the Eppo environment
+- **Subject Key**: A unique identifier for the subject (usually a user ID)
+
+Each assignment requires:
+
+- **Flag Key**: Identifies which set of configuration rules to use
+- **Default Value**: Fallback value if assignment fails or rules don't match
+
+```dart
+// Make an assignment
+final String variation = Eppo.getStringAssignment(
+  'my-neat-feature',
+  'default-value',
+);
+```
+
+### Assignment Types
+
+The SDK provides different assignment functions based on the type of value you need:
+
+| Function                 | Return Type            |
+| ------------------------ | ---------------------- |
+| `getBooleanAssignment()` | `bool`                 |
+| `getIntegerAssignment()` | `int`                  |
+| `getNumericAssignment()` | `num`                  |
+| `getStringAssignment()`  | `String`               |
+| `getJSONAssignment()`    | `Map<String, dynamic>` |
+
+### Using Assignments
+
+After receiving an assignment, your application should implement logic to modify the user experience accordingly:
+
+```dart
+final String variation = Eppo.getStringAssignment(
+  'my-neat-feature',
+  'default-value',
+);
+
+// Render different components based on assignment
+switch(variation) {
+  case 'landing-page-a':
+    return renderLandingPageA();
+  case 'landing-page-b':
+    return renderLandingPageB();
+  default:
+    return renderLandingPageC();
+}
+```
+
+## Experiments
+
+While feature flags are useful, they do not send you any information about how your users are interacting with the feature. 
+Experiments provide a way to collect data about these interactions using whichever logging and data warehousing system you prefer.
```

Comment:
Thinking here we want something like experiments use randomization to compare users to derive impact. ALthough I'm also fine leaving this all as-is and having Niko or somebody give it a once over.

Reply:
BTW I copied this from another page - so this copy is all over the place